### ISSUE TITLE
Error with 'screenfetch' and 'xorg-xdpyinfo' -- cannot resolve 'libdmx'

### ISSUE THREAD

**Issue by jrodd96** [AnarchyLinux/installer#934]
## Basic info:
Attempting to install Anarchy Linux to Dell desktop PC -- GNOME Desktop w/ encrypted LVM; UEFI

**Anarchy version**: v1.0.8

**Installation method (VM/Host)**: ISO of installer on USB to 1 TB HDD on Dell desktop PC; Ethernet

**System specifications (Processor, GPU, RAM ...)**:
Dell Inspiron 3670 w/ Intel i5-8400 CPU
Intel UHD Graphics 630 GPU
12 GB DDR4 RAM
1 TB Seagate HDD

## What you expected would happen

Was attempting to install Anarchy Linux to my Dell PC using the ISO installer on USB:
- Updated the installer after boot
- followed installer options to enable UEFI, set swap space, and use encrypted LVM (btrfs) for root partition
- chose Desktop Quick Install option w/ GNOME as desktop manager
- no additional packages selected
## What actually happened
Install immediately fails after confirming install with message:
`Error: Failed to install packages to new root`

## Extra info

Terminal output immediately after exit of installer script is as follows:
```
:: The following packages cannot be upgraded due to unresolvable dependencies:
       screenfetch  xorg-xdpyinfo

:: Do you want to skip the above packages for this upgrade? [y/N] error: failed to prepare transaction (could not satisfy dependencies)

:: unable to satisfy dependency 'libdmx' required by xorg-xdpyinfo
:: unable to satisfy dependency 'xorg-xdpyinfo' required by screenfetch
:: unable to satisfy dependency 'libdmx' required by xorg-xdpyinfo
==> ERROR: Failed to install packages to new root
2019-12-19 19:50 : Install failed: please report to developer
```
Even disabling screenfetch in /etc/anarchy.conf doesn't fix the error because `xorg-xdpyinfo` is apparently a required dependency for something xorg-related.

## Bug report checklist

* [ x] I've read the [Reporting Issues Wiki Page](https://github.com/AnarchyLinux/installer/wiki/Reporting-issues)
* [ x] I've removed unnecessary sections of the bug submission template
* [ x] I understand that if I don't do everything listed in the checkmarks above my issue won't be considered a priority and may get closed at any time without warning, because I didn't take the time to properly submit my bug report

**Comment by crian** [comment]
'libdmx' was removed form the Arch repos and 'xorg-xdpyinfo' hasn't been updated to a newer version without the dependency. Newer version is currently only available in the testing repo.

**Comment by erazemkokot** [comment]
Fixed in [6fb243213203691727ceeec52bf1098136ab9375](https://github.com/AnarchyLinux/installer/commit/6fb243213203691727ceeec52bf1098136ab9375)

**Comment by jrodd96** [comment]
> (By removing screenfetch, since xorg-xdpyinfo is broken atm)

Not sure if there was additional configurations I needed to do, but I state in my report that I went in and removed screenfetch from /etc/anarchy.conf and tried the install again and even tried rebooting and trying again, but the error for some reason persisted. The only difference in the error is it doesn't mention screenfetch in the error just xorg-xdpyinfo:

```
:: The following packages cannot be upgraded due to unresolvable dependencies:
       xorg-xdpyinfo

:: Do you want to skip the above packages for this upgrade? [y/N] error: failed to prepare transaction (could not satisfy dependencies)

:: unable to satisfy dependency 'libdmx' required by xorg-xdpyinfo
==> ERROR: Failed to install packages to new root
2019-12-20 15:10 : Install failed: please report to developer
```
